src/main/java/bookstore/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package bookstore.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
extend it from RuntimeException and you won't have to specify in each place of your code `throws RegistrationException`